cmake/third_party/openfst.cmake

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8.2)
+project(openfst-download NONE)
+
+include(ExternalProject)
+ExternalProject_Add(openfst
+    GIT_REPOSITORY https://github.com/kkm000/openfst
+    GIT_TAG winport

```

Comment:
You would need to use the full hash actually, not the git tag, so that cmake doesn't have to ping the server on every rebuild to make sure that the remote's definition of the hash hasn't changed. (Meanwhile, hashes are immutable.)

Reply:
I am still wondering why there is no mirror in kalid-asr project namespace.

Reply:
Updated to use a tag.

Reply:
RE the mirror: @kkm000  is a Kaldi contributor so we don't have to worry about him changing his OpenFST repo in a way that breaks stuff.  And Kaldi install is already dependent on github being up, so we don't need to worry about that being down (plus it's super reliable).